magento/magento2
# Keep dependencies in composer.json sections sorted alphabetically

### Description (*)


Since we have the [sort](https://github.com/magento/magento2/blob/2.4-develop/composer.json#L11) flag set to **true**, all sections must be sorted alphabetically.

### Manual testing scenarios (*)

1. open **composer.json** file
2. check the [replace](https://github.com/magento/magento2/blob/2.4-develop/composer.json#L106-L334) section

### Questions or comments

All [package links](https://getcomposer.org/doc/04-schema.md#package-links) sections should be sorted this way:
- [conflict](https://getcomposer.org/doc/04-schema.md#conflict)
- [replace](https://getcomposer.org/doc/04-schema.md#replace)
- [provide](https://getcomposer.org/doc/04-schema.md#provide)
- [suggest](https://getcomposer.org/doc/04-schema.md#suggest)

### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#31182: Keep dependencies in composer.json sections sorted alphabetically
